services/payments/src/simcore_service_payments/models/schemas/acknowledgements.py

```diff
@@ -19,6 +19,15 @@ class _BaseAckPayment(_BaseAck):
     invoice_url: HttpUrl | None = Field(
         default=None, description="Link to invoice is required when success=true"
     )
+    # NOTE: Why invoice_pdf, stripe_invoice_id and stripe_customer_id nullable? Currently, we are dependent on a third party that is making
```

Comment:
This description is useful but descriptions as  "Stripe invoice ID" for a variable called `strip_invoice_id` is simply redundant. I would rather avoid a description if there everything is clear by the context

Reply:
The value in this comment is that in theory, these fields should be mandatory, but I wrote the reason why I didn't make it mandatory for now.